recipes/xvfbwrapper/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.2.8" %}
+
+package:
+  name: xvfbwrapper
+  version: {{ version }}
+
+source:
+  fn: xvfbwrapper-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/x/xvfbwrapper/xvfbwrapper-{{ version }}.tar.gz
+  md5: c8ce7c35fc555a6af272e7e17cf0b5c0
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I guess it is optional based on the code I saw. Would it be alright if we add `setuptools` to the build requirements? I think this should pass after that change.